semgrep-core/tests/OTHER/rules/sym_prop_no_merge.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: test
+  options:
+    symbolic_propagation: true
+  pattern-either:
+    - pattern: return a.foo
```

Comment:
would be nice though that at some point we allow $X.foo to match!
That is when we do the merge of a.foo and b.foo we generalize to $X.foo on the target side!